test/hotspot/gtest/metaspace/test_is_metaspace_obj.cpp

```diff
@@ -49,7 +49,7 @@ class MetaspaceIsMetaspaceObjTest {
   }
 
   void do_test(Metaspace::MetadataType mdType) {
-    _lock = new Mutex(Monitor::leaf, "gtest-IsMetaspaceObjTest-lock", false, Monitor::_safepoint_check_never);
+    _lock = new Mutex(Monitor::leaf, "gtest-IsMetaspaceObjTest-lock", Monitor::_safepoint_check_never);
```

Comment:
Again unclear what impact false to true might have here - assuming VM threads even lock this? Ditto for other metaspace tests.

Reply:
It has zero impact.  These locks just want no safepoint checking.